src/Core/Geographic/Extent.js

```diff
@@ -10,14 +10,33 @@ import CRS from 'Core/Geographic/Crs';
 const _dim = new THREE.Vector2();
 const _dim2 = new THREE.Vector2();
 const _countTiles = new THREE.Vector2();
+const _box = new THREE.Box3();
 const tmsCoord = new THREE.Vector2();
 const dimensionTile = new THREE.Vector2();
 const defaultScheme = new THREE.Vector2(2, 2);
 const r = { row: 0, col: 0, invDiff: 0 };
 
+const cNorthWest =  new Coordinates('EPSG:4326', 0, 0, 0);
+
 const southWest = new THREE.Vector3();
 const northEast = new THREE.Vector3();
 
+
+/**
+ * The method projects the box limits in WGS84 projection
+ *
+ * @param      {string}  crs     The data box crs projection
+ * @param      {THREE.Box3}  box     The box to project to WGS84
+ * @return     {THREE.Box3}  the projected box.
+ */
+function box3ProjectedtoWGS84(crs, box)  {
+    cNorthWest.crs = crs;
+    cNorthWest.setFromVector3(box.min).as('EPSG:4326', cNorthWest).toVector3(box.min);
```

Comment:
Could you rename to `corner` or create two `Coordinates` : one for the south-west corner and another one for the north-east corner ?

Reply:
done